### ISSUE TITLE
Decide on user-friendly configuration cell inclusion syntax

### ISSUE THREAD

**Issue by cos** [kframework/k-legacy#1897]
See comment here for details #1873

**Comment by daejunpark** [comment]
I think we can keep both, considering `<s/>` as a syntactic sugar. In most cases, `<s/>` will be sufficient, which is succinct than `initSCell(Init)` and doesn't require user to understand the initialization mechanism. But there should be a situation that requires a fine controls, which will need `initSCell` approach.

**Comment by cos** [comment]
I agree we should keep both, at least for now. I just added the issue because @dwightguth raised the concern and @grosu said he wants a discussion to make a final decision. We'll get back to it when we have time.
